nat-lab/natlab.py

```diff
@@ -41,6 +41,14 @@ def start():
             "LIBTELIO_ENV_NAT_LAB_DEPS_TAG": LIBTELIO_ENV_NAT_LAB_DEPS_TAG,
         },
     )
+
+    if "GITLAB_CI" in os.environ:
+        with open("docker-compose.yml", "r", encoding="utf-8") as file:
+            filedata = file.read()
+        filedata = filedata.replace('ports: ["58001"]', "ports: []")
```

Comment:
Would be nice if you add some check for 'ports: ["58001"]' to catch any modifications to it, and fail it nat-lab run with a meesage

Reply:
I added a check with a `RuntimeError` in case the port mapping is not there